### ISSUE TITLE
Disable auto-retry on "You have to relog in order to place a bet" error

### ISSUE THREAD

**Issue by llamasoft** [ncla/LoungeDestroyer#285]
This may be a non-issue, but if the auto-retry feature (at least for placing bets) ever encounters the error "You have to relog in order to place a bet" then it can never be recovered from.  

If you relog in without closing the auto-retry window it will still continue to fail because the `tlss` POST value it's attempting to use has been invalidated.  Attempting to retry will continually result in failure.

**Comment by ncla** [comment]
We handled this re-log issue in the past because Lounge was throwing this error due to expired CSRF tokens, that was fixed by fetching the match page again and replacing the CSRF token. This was later however removed.

The re-log error can appear in other cases as well.

I don't think we want to handle it by disabling auto-betting because that would allow CSGOLounge to exploit the error message to disable auto-betting (if they ever want to).

**Comment by llamasoft** [comment]
Good point about exploiting the error.  

You mentioned that in the past you fetched the match page again to update the CSRF token.  Why was that functionality disabled?  Was it causing issues?  (Is it anything I can help with?)

**Comment by ncla** [comment]
That was really long time ago, before Chrome extension was published. It might have been a bug on the site which caused the CSRF to expire. I don't know exactly why I removed it, but most likely because it was a bloat/unnecessary handling in the code.

**Comment by llamasoft** [comment]
Fair enough.  If I can put together a workable solution (that's not too bloated) then I'll submit a pull request.
Thanks for the quick responses!

**Comment by ncla** [comment]
When do you encounter this error though? I don't see any reason to handle re-log errors unless something on their back-end is causing it (and not user).

**Comment by llamasoft** [comment]
I'm not exactly how I triggered the issue, but it involved being signed in on two devices.

I was signed in on device A then later in the day signed on device B.  I later went back on device A and still appeared to be logged in (opening a new match page still showed my returns and allowed me to attempt to place a bet).  However, when I attempted to place a bet, that's when I encountered the issue.

I'm not sure if I would call that a user error for signing in on two devices or a site error for apparently leaving me logged in on the first device.  Either way, it appears to be an edge case and I need to do some more testing to see if I can reproduce the issue.
